infra/base-images/base-runner/Dockerfile

```diff
@@ -87,6 +88,15 @@ RUN wget https://repo1.maven.org/maven2/org/jacoco/org.jacoco.cli/0.8.7/org.jaco
 COPY install_javascript.sh /
 RUN /install_javascript.sh && rm /install_javascript.sh
 
+# Copy built ruby and ruzzy from builder
```

Comment:
Do you know how much this adding?

Reply:
```
139M	/usr/local/rvm
140M	/install/ruzzy
```

If it's important, I can try to reduce it (but without promises).

Reply:
eh this is a little big. We can have it for now, but if you can do any cheap fixes it would help.
(size makes things harder for CIFuzz which needs to pull images quickly and has limited disk space.

Reply:
I should be able to limit it to ~140M, but going below it may be challenging (not impossible, most likely).

Reply:
Let's try if we can please.

Reply:
We can avoid creating `/install/ruzzy` and instead install in the default directory (somewhere in `/usr/local/rvm`).
That helps to save around 100MB.

Followup: https://github.com/google/oss-fuzz/pull/12148#discussion_r1672995348